api/README.md

```diff
@@ -22,7 +22,10 @@ We currently provide 6 file types:
 │   ├── county/
 |   |   ├── counties_top_100.json
 │   │   ├── <state abreviation>.<fips>.<intervention number>.json
+│   │   ├── <fips>.<intervention number>.json
```

Comment:
Can you update these changes to match the new directory structure (should be `us |- counties |- <fips>.<intervention>.json` or similar)?